Extension/YapoSubModule.cs

```diff
@@ -25,7 +25,8 @@ protected override void OnSubModuleLoad()
                 FileDatabase.Initialise(Strings.MODULE_FOLDER_NAME);
                 Settings settings = FileDatabase.Get<Settings>(Settings.InstanceId) ?? new Settings();
                 SettingsDatabase.RegisterSettings(settings);
-                
+                FileDatabase.SaveToFile(Strings.MODULE_FOLDER_NAME, settings);
```

Comment:
I don't think this is necessary. From the ModLib documentation settings are stored automatically when they are edited by the user in the main menu. The only other time to save settings would be when we change them programmatically, which isn't performed here.

Reply:
I wasn't aware the mod saves its settings off file. However the mod still has settings that aren't covered in the settings screen. So that's the main issue for now.

I'm gonna try and think of a way to show settings in a more advanced section of our mod so the save to file is unnecesary

Reply:
Will keep this in for now. At most I think it will just write to settings on load up